opencti-platform/opencti-front/src/private/components/settings/decay/DecayRulePopover.tsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState } from 'react';
+import { graphql, useMutation } from 'react-relay';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import Button from '@mui/material/Button';
+import IconButton from '@mui/material/IconButton';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import MoreVert from '@mui/icons-material/MoreVert';
+import makeStyles from '@mui/styles/makeStyles';
+import { useNavigate } from 'react-router-dom-v5-compat';
+import { DecayRule_decayRule$data } from './__generated__/DecayRule_decayRule.graphql';
+import { decayRuleEditionMutation } from './DecayRuleEdition';
+import Transition from '../../../../components/Transition';
+import { useFormatter } from '../../../../components/i18n';
+import { handleError } from '../../../../relay/environment';
+
+const useStyles = makeStyles(() => ({
```

Comment:
`makeStyles` is deprecated. Try to put style directly in JSX using `style={{...}}` for html elements or `sx={{...}}` for MUI elements